doc/cjdns.md

```diff
@@ -0,0 +1,92 @@
+# CJDNS support in Bitcoin Core
+
+It is possible to run Bitcoin Core over CJDNS, an encrypted IPv6 network that
+uses public-key cryptography for address allocation and a distributed hash table
+for routing.
+
+## What is CJDNS?
+
+CJDNS is like a distributed, shared VPN with multiple entry points where every
+participant can reach any other participant. All participants use addresses from
+the `fc00::/8` network (reserved IPv6 range). Installation and configuration is
+done outside of Bitcoin Core, similarly to a VPN (either in the host/OS or on
+the network router).
+
+Compared to IPv4/IPv6, CJDNS provides end-to-end encryption and protects nodes
+from traffic analysis and filtering.
+
+Used with Tor and I2P, CJDNS is a complementary option that can enhance network
+redundancy and robustness for both the Bitcoin network and individual nodes.
+
+Each network has different characteristics. For instance, Tor is widely used but
+somewhat centralized. I2P connections have a source address and I2P is slow.
+CJDNS is fast but does not hide the sender and the recipient from intermediate
+routers.
+
+## Installing CJDNS and connecting to the network
+
+To install and set up CJDNS, follow the instructions at
+https://github.com/cjdelisle/cjdns#cjdns.
+
+Don't skip steps
+["2. Find a friend"](https://github.com/cjdelisle/cjdns#2-find-a-friend) and
+["3. Connect your node to your friend's
+node"](https://github.com/cjdelisle/cjdns#3-connect-your-node-to-your-friends-node).
+You need to be connected to the CJDNS network before it will work with your
+Bitcoin Core node.
+
+Typically, CJDNS might be launched with `sudo ./cjdroute < cjdroute.conf` and it
+sheds permissions after setting up the [TUN](https://en.wikipedia.org/wiki/TUN/TAP)
+device. You may also [launch it as an unprivileged
+user](https://github.com/cjdelisle/cjdns/blob/master/doc/non-root-user.md) with
+some additional setup.
+
+The network connection can be checked with `./tools/peerStats`.
```

Comment:
nit: here and above in `./cjdroute` command - could that be misinterpreted as `tools/peerStats` being in the Bitcoin Core source repository? Maybe something like `$CJDNS/tools/peerStats` would be more clear?

Reply:
updated, thanks